arq/utils.py

```diff
@@ -1,8 +1,12 @@
 import asyncio
 import logging
+import os
+import pytz
```

Comment:
pytz is not a dependency of this package, and we're not adding it.

You need to catch `ImportError` and only check for other timezones if it's installed.